E3SM-2-0/piControl_r1i1p1f1.json

```diff
@@ -0,0 +1,105 @@
+{
+  "#note_source_type": "explanation of what source_type is goes here",
+
+  "source_type": "AOGCM AER",
+
+  "#note_experiment_id": "CMIP6 valid experiment_ids are found in CMIP6_CV.json",
+
+  "experiment_id": "piControl",
+
+  "activity_id": "CMIP",
+
+  "sub_experiment_id": "none",
+
+  "realization_index": "1",
+
+  "initialization_index": "1",
+
+  "physics_index": "1",
+
+  "forcing_index": "1",
+
+  "#note_run_variant": "Text stored in attribute variant_info (recommended, not required description of run variant)",
+
+  "run_variant": "",
+
+  "parent_experiment_id": "piControl",
```

Comment:
Based on the error in @TonyB9000 's test, `"parent_experiment_id": "piControl-spinup"`